pages/streatview.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div>テンプレート{{test}}</div>
+</template>
+<script>
+export default {
+  head() {
+    return {
+      title: '',
+    }
+  },
+  layout: 'default',
+  components: {},
+  middleware: [],
+  data() {
+    return {
+      test: 3,
+      corArr: []
+    }
+  },
+  watch: {},
+  computed: {
+    google() {
+      return window.google
+    }
+  },
+  created() {},
+  beforeMount() {},
+  mounted() {
+    this.getStreatViewCordination()
+  },
+  beforeDestroy() {},
+  methods: {
+    // 引数としてやまりょうから店舗の位置情報を受け取る,google, origin, destination
+    getStreatViewCordination() {
+      const origin = "東京都庁"
+      const destination = "茨城県庁"
+      var directionsService = new google.maps.DirectionsService()
+      var request = {
+          // 開始地点
+          origin: origin,
+          // 終了地点
+          destination: destination,
+          // 移動方法：徒歩
+          travelMode: google.maps.DirectionsTravelMode.WALKING,
+          // 残り距離の表示がキロメートルになる
+          unitSystem: google.maps.DirectionsUnitSystem.METRIC,
+          // 
+          optimizeWaypoints: true,
+      }
+      // thisのスコープの都合上コールバック関数ないでdataを使用するために代入する
+      var corArr = this.corArr
+      // requestを使ってルート検索
+      var result = directionsService.route(request, function(result, status) {
```

Comment:
```suggestion
       var result = directionsService.route(request, (result, status) => {
```
アロー関数にすると，コールバック関数内でも data の変数を this. で使用できるようになると思うよ～！